README.md

```diff
@@ -2,15 +2,35 @@
 
 ## quickstart
 
-Install snakemake. Then run:
+1. [Install `conda`](https://bioconda.github.io/user/install.html#install-conda)
+2. Set up channels
+```
+conda config --add channels defaults
+conda config --add channels bioconda
+conda config --add channels conda-forge
+```
+
+3. Clone this repository (`git clone https://github.com/NCBI-Codeathons/bothie`)
+4. Create the `conda` environment:
 
 ```
-./runme.py sourmash/conf-basic.yml sourmash data/genome-s10.fa.gz
+cd bothie
+conda env create -f environment.yaml
 ```
 
-## required software
+5. Activate the `conda` environment:
+```
+# To activate this environment, use
+#
+#     $ conda activate spacegraphcats
```

Comment:
hah, the name is because I reused the environment.yml from [spacegraphcats](https://github.com/spacegraphcats/spacegraphcats/).

We could change the name in `environment.yml` to `bothie`, and/or we could change the `conda env create` to give a name with e.g. `-n bothie`.

Reply:
I like the idea of adding `-n bothie` . I can remove the name from `environment.yml`.
`spacegraphcats` is a great name!